components/profile/overview/regularity/regularity.tsx

```diff
@@ -0,0 +1,100 @@
+import Image from 'next/image'
+import { TProfileOverview } from 'types/profile'
+
+import Chart from '@/icons/Chart.svg'
+import TrendUpTshirt from '@/icons/TrendUpTshirt.svg'
+
+type RegularityType = TProfileOverview['regularity']
+
+interface IProps extends RegularityType {
+  name: string
+  clubLogo: string
+}
+
+export const Regularity = ({
+  fromBench,
+  ofMatches,
+  firstEleven,
+  bench,
+  outsideCadre,
+  name,
+  clubLogo,
+}: IProps) => {
+  const getPercents = (value: number) => Math.round((value / ofMatches) * 100)
+  const regularityCondition = firstEleven > fromBench + bench
+
+  return (
+    <div>
+      <div className="mt-7 mb-8 flex items-center gap-7 bg-white p-5 pl-7 shadow-default">
+        {regularityCondition ? (
+          <TrendUpTshirt className="icon-40" />
+        ) : (
+          <Image
+            src={clubLogo}
+            width={40}
+            height={40}
+            alt={name}
+            className="icon-24"
+          />
```

Comment:
```suggestion
          <Image
            src={additional.teamLogo}
            width={40}
            height={40}
            alt={name}
            className="icon-24"
          />
```
I'll fix the name later to have Url at the end

Reply:
oh i didnt notice it